core/sequencer/sequencer_api.proto

```diff
@@ -75,8 +68,29 @@ message DefineRevisionsRequest {
 
 // DefineRevisionResponse contains information about freshly defined revisions.
 message DefineRevisionsResponse {
-  // outsanding_revisions a list of all the defined revisions which are not yet applied.
-  repeated int64 outstanding_revisions = 1;
+  // unapplied_begin is the first outstanding revision (inclusive).
+  int64 unapplied_begin = 2;
+  // unapplied_end is the last outstanding revision (exclusive). It is equal to
+  // unapplied_begin if there are no outstanding revisions.
+  int64 unapplied_end = 3;
+  // Deprecated tags, do not reuse.
+  reserved 1;
+}
+
+// GetUnappliedRevisionsRequest contains information needed to query the set of
+// outstanding revisions.
+message GetUnappliedRevisionsRequest {
+  // directory_id is the directory to examine the outstanding revisions for.
+  string directory_id = 1;
+}
+
+// GetUnappliedRevisionResponse contains the set of outstanding revisions.
+message GetUnappliedRevisionsResponse {
+  // begin is the first outstanding revision (inclusive).
+  int64 begin = 1;
```

Comment:
Nit: these names should match the names in DefineRevisionsResponse

Reply:
Got back to lists of revisions.